enterprise/admin/__init__.py

```diff
@@ -633,6 +634,59 @@ def get_form(self, request, obj=None, **kwargs):
         form.base_fields['content_filter'].initial = json.dumps(get_default_catalog_content_filter())
         return form
 
+    def get_actions(self, request):
+        """
+        Disallow the delete selected action as that does not send a DELETE request to enterprise-catalog
+        """
+        actions = super(EnterpriseCustomerCatalogAdmin, self).get_actions(request)
+        if 'delete_selected' in actions:
+            del actions['delete_selected']
+        return actions
+
+    def save_model(self, request, obj, form, change):
+        """
+        On save, creates or updates the corresponding catalog in the enterprise-catalog IDA.
+
+        `change` indicates whether or not the object is being updated (as opposed to created).
+        """
+        # Save the changes no matter what as we can copy them over later if we need to
+        super(EnterpriseCustomerCatalogAdmin, self).save_model(request, obj, form, change)
+
+        catalog_uuid = obj.uuid
+        catalog_client = EnterpriseCatalogApiClient(user=request.user)
+
+        if change:
+            update_fields = {
+                'enterprise_customer': str(obj.enterprise_customer.uuid),
+                'title': obj.title,
+                'content_filter': obj.content_filter,
+                'enabled_course_modes': obj.enabled_course_modes,
+                'publish_audit_enrollment_urls': obj.publish_audit_enrollment_urls,
```

Comment:
something i changed because it appeared to break things and seemed to work was to convert this from a boolean to a string, i got a bad request exception when just passing it directly. if you haven't run into that in your testing, then that's cool, just calling it out.

Reply:
Huh interesting. I haven't seen that, but I'll see if I can replicate